### DISCUSSION TITLE
Website development in Persian language

### DISCUSSION THREAD

**Discussion by mobin-ux** [tailwindlabs/tailwindcss#3716] [Help]
Hello, my name is Mobin
I want to change the site and document of this framework to Persian.
With a domain separate from the main domain that I will own.
I also want to translate the templates on https://tailwindui.com/components into Persian and sell them.
I also want to record tutorials and sell some of them and publish others for free

Persian language is for Iran and all payments are made in Iranian currency (Rials)

Do I have such permission ???
I do not violate the licenses ??

**Comment by adamwathan** [comment]
Hey! It is absolutely not okay to translate Tailwind UI templates and sell them. Please see the license here:

https://tailwindui.com/license

Tailwind UI is a copyrighted commercial product and no one has the right to sell that intellectual property except us. You are welcome to design, build and sell your own templates completely from scratch, but you can't resell our templates.

Our documentation + the documentation website itself is also not licensed for modification or redistribution (which is why our documentation repo has no license file) so unfortunately that is not permitted either.

You are totally welcome to create and sell educational content like video tutorials though 👍🏻

**Reply by mobin-ux** [reply]
:) Thanks
But I did not notice a few cases.
So I can not even change the appearance of tailwindcss.com and use it to introduce the framework?
Or I can not even change the framework educational document in Persian https://tailwindcss.com/docs ??

:(

**Comment by mobin-ux** [comment]
@adamwathan pls aswer :)

**Comment by redspiderweb1** [comment]
[Website development Dubai](https://www.redspider.ae/service/web-design-development/) has become highly competitive, especially with the growing demand for multilingual websites, including Persian (Farsi) language support. Businesses in Dubai often target diverse audiences, so creating websites in Persian helps connect with Iranian and Farsi-speaking customers effectively. Professional web development services in Dubai focus on modern design, mobile responsiveness, and fast performance to ensure a smooth user experience. SEO optimization is also essential to rank higher in search engines and attract more organic traffic. Developers typically use advanced technologies like WordPress, Laravel, and React to build scalable and secure websites. A well-developed Persian-language website can significantly improve brand visibility and expand business reach in both local and international markets.
